//line :1
package hPMrClpC

import (
	testlog "X0MraxO81Me"
	"runtime"
	syscall "bUKeamF"
)


var USSgbG96n []string

func init() {
	if runtime.GOOS == "windows" {

		return
	}
	USSgbG96n =  /*line :1*/ kvYaar24hG()
}

func kvYaar24hG() []string




func ODlau62QrT9c() int	{ return  /*line :1*/ syscall.OeMe9jE() }




func CJ9SzZOTa2J4() int	{ return  /*line :1*/ syscall.ZNXmxC() }




func LIiiwAt() int	{ return  /*line :1*/ syscall.ZA1sFx5_XxZX() }




func G_CPThWexL() int	{ return  /*line :1*/ syscall.BDGfBD() }





func MJ0TyiGqpkS() ([]int, error) {
	y2rVef_lI, w50uhPri :=  /*line :1*/ syscall.WafuobroC()
	return y2rVef_lI,  /*line :1*/ BTaHHl("getgroups", w50uhPri)
}






func U9KBmNojTul(aGXbi7Dvax int) {
	if aGXbi7Dvax == 0 &&  /*line :1*/ testlog.CeWroRg() {

		 /*line :1*/ panic("unexpected call to os.Exit(0) during test")
	}

	 /*line :1*/ uAGeW9rBH(aGXbi7Dvax)

	 /*line :1*/ syscall.QcqSaNcYkFmT(aGXbi7Dvax)
}

func uAGeW9rBH(hv02FJC int)
